tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -432,11 +432,7 @@ TRTEngineOp::TRTEngineOp(OpKernelConstruction* context)
   // 2.0.
   calibration_mode_ =
       (use_calibration_ && precision_mode_ == TrtPrecisionMode::INT8 &&
-       calibration_data.empty());
-  if (!calibration_data.empty()) {
-    calibrator_.reset(new TRTInt8Calibrator(calibration_data));
-    calibration_data.resize(0);
-  }
```

Comment:
While I am still trying to understand the implication of these few line of change, have you tested TF1 with calibration?
calibartion_data attribute is  also used for TF1, that is in the V1 converter.

Reply:
I can see how this can be an issue. I haven't tested with TF1, so I'll have to ask around on how to do that.

Reply:
I'm still having issues building with TF1, but what do you think of changing this such that we still load the calibrator from the calibration_data attribute if the V1 converter was used and we only load the calibrator from the TRTEngineCacheResource if the V2 converter was used.

Reply:
right, that is what we should do.